### PR TITLE
[FEAT] Increase Fish Hitbox

### PR DESCRIPTION
**Pull request by c-py**

### CODE REVIEW

File: src/features/island/fisherman/FishingChallenge.tsx [hunk 1]
```diff
@@ -159,27 +159,21 @@ export const FishingChallenge: React.FC<Props> = ({
...
+    const tentacleAngle = extractAngle(tentacleProps.transform.get());
 
     const degreeDifference = Math.abs(
       ((normaliseAngle(greenBarAngle) - normaliseAngle(tentacleAngle) + 180) %
         360) -
         180
     );
 
-    const fishBuffer = 5;
+    const fishBuffer = 10;
```
Comment (c-py) [inline src/features/island/fisherman/FishingChallenge.tsx:170]: Fish is approx 20px, so this is half of the fish buffer on either side of the green line.

File: src/features/island/fisherman/FishingChallenge.tsx [hunk 2]
```diff
@@ -127,7 +127,7 @@ export const FishingChallenge: React.FC<Props> = ({
     from: { transform: `rotate(${castFrom}deg)` },
     to: { transform: `rotate(${castTo}deg)` },
     onRest: () => {
-      if (misses >= 2) {
+      if (misses >= 2 && hits < health) {
```
Comment (c-py) [inline src/features/island/fisherman/FishingChallenge.tsx:130]: Small bug fix where `onRest` was triggering a failure, after a winning game won on 2/3 misses.

File: src/features/island/fisherman/FishingChallenge.tsx [hunk 3]
```diff
@@ -154,32 +154,29 @@ export const FishingChallenge: React.FC<Props> = ({
     },
   });
 
-  const barHeight = 10;
+  const greenBarPercentage = 10;
+  // Half bar (bar angle is 10% of 360deg).
+  const halfBarDegrees = 360 / greenBarPercentage / 2;
+  // Add 5% tolerance to be more forgiving
+  const halfBarDegreesWithTolerance = halfBarDegrees * 1.05;
```
Comment (c-py) [inline src/features/island/fisherman/FishingChallenge.tsx:161]: 18.9% tolerance

File: src/features/island/fisherman/FishingChallenge.tsx (hunk 2 above)
Comment (adamhannigan) [reply src/features/island/fisherman/FishingChallenge.tsx:130]: OMG! 😱 

That got me so many times 😆

**Review by adamhannigan — APPROVED** [review]
LGTM! 🚀

**Pull request merged**